buzzmobile/sense/inputer/inputer.lisp

```diff
@@ -0,0 +1,7 @@
+(in-package :buzzmobile)
+
+(defun inputer ()
+    (with-ros-node ("inputer")
+        (let ((i 0) (pub (advertise "destination" "std_msgs/String")))
+            (loop-at-most-every .1
+                (publish-msg pub :data (read))))))
```

Comment:
You are not handling any of the errors `(read)` can throw.
http://www.lispworks.com/documentation/HyperSpec/Body/f_rd_rd.htm